src/gg.regression.unity.bots/Runtime/Scripts/RemoteOrchestration/Models/ActiveSequence.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Text;
+using RegressionGames.StateRecorder.BotSegments.Models;
+using RegressionGames.StateRecorder.JsonConverters;
+using UnityEngine.Animations;
+
+// ReSharper disable InconsistentNaming
+namespace RegressionGames.RemoteOrchestration.Models
+{
+    [Serializable]
+    public class ActiveSequence
+    {
+        public string name;
+        public string resourcePath;
```

Comment:
Should also have `description` here, for the webapp's sake. 9 times out of 10, I expect the active sequence to match one from the availableSequences list one-to-one, but we've mentioned that users can change the name and description after registration, which won't be reflected in that list

Reply:
I've been thinking about how the webapp will respect discrepancies between the registered `availableSequences` and sequences that are modified post-registration. If we're viewing the clients at some moment in time, then obviously we can show trust the active sequence as the "correct version". But when we schedule a workAssignment for a client, we won't know about any post-registration changes to that list, meaning any historical view of completed assignments won't match up with what the dashboard says the client actually ran.

This might be a nothing-burger / future issue. But we could send the `availableSequences` in the heartbeat to keep them up-to-date. A WorkAssignment would then capture the entire sequence object from the available list at the time it's scheduled, so we get a more accurate snapshot of what "version" the client actually ran for that assignment.

Reply:
I think nothing-burger.  In order for you to get one after registration.  You'd have a to create a new sequence in a game instance already running with -rgremoteworker.  This would mean that a user is manually controlling a system meant for automation.  And not just controlling it, using it as a live dev system.  I think it will be fine for any new sequence not to show up in the remote console until the next game play restart

Reply:
As for description, I can add this in I guess, but currently I don't see any UI flows that show anything other than the name and path.

Reply:
Bringing up adding/removing was sort of tangential, I apologize. 

The core of my point is... 
why are we passing an object instead of just the resourcePath? -> Because we said the name (and description) could change. 
What does that mean for the UI? -> Viewing active clients will show the "correct" value based on `activeSequence` but a client's `WorkAssignment`s only have visibility to the list. Thus if the `availableSequences` list never reflects changes to name/description then the `WorkAssignment` be forced to display "inaccurate" information (ie. the name and description of the sequence at time of registration rather than after being modified post-registration).

Does this clarify what I was trying to get at?

Reply:
- As for passing an object.  I was doing this mostly to clarify the case where something that is NOT a BotSequence (segmentZip/segmentList/segment/etc) is running.  Since that will NOT have a resourcePath, but WILL have an activeSequence entry in the status so that the server realizes something is running

```csharp
                // else - a zip file or other bot segments are running outside of a sequence
                return new ActiveSequence()
                {
                    name = "BotSegments are active outside of a BotSequence",
                    resourcePath = "",
                    sequence = null
                };
```
- "(ie. the name and description of the sequence at time of registration rather than after being modified post-registration)"  ... this is what i just tried to respond to above.  Modifications post registration on one of these systems are incredibly unlikely and if so, not a big deal to not be reflected until next start of the game.

Reply:
`description` added

Reply:
Ah okay I understand now, thank you